### ISSUE TITLE
PDF download of fair use scans

### ISSUE THREAD

**Issue by eroux** [buda-base/public-digital-library#799]
One user was surprised that the PDF they downloaded of a fair use work didn't contain all the pages. I think we're quite used to that and users from the legacy website are too, but I'm thinking perhaps we should change the system a little bit for two reasons:
- it probably doesn't make a lot of sense for new users
- since the trick to download the full PDF doesn't work on BUDA anymore, it's not really useful to users of the legacy site

There are several ways to think of this:
- we could just not allow download of PDF if the result would be the first and last 20 pages (since it's not that useful anyway)
- we could change the wording a little bit and use "download extract" instead of "download"

wdyt @JannTibetan @karmagongde ?

**Comment by karmagongde** [comment]
Hi Elie, 
Sometimes the first front 20 pages are useful to look for the Table of Contents. So, changing the working would be better.

**Comment by eroux** [comment]
the consensus seems to be a popup with the same wording as the text that is in the middle of the page ("This work is copyrighted by a third party...") before the PDF can be downloaded

**Comment by JannTibetan** [comment]
Yes. Let's give them a reminder that they are being a given a preview of the text according to what is allowable under the doctrine of "fair use." BDRC is not set up to offer more than that, but Internet Archive is.
